packages/ppx_rapper/ppx_rapper.3.0.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Syntax extension for Caqti/PostgreSQL queries"
+maintainer: ["Roddy MacSween <github@roddymacsween.co.uk>"]
+authors: ["Roddy MacSween <github@roddymacsween.co.uk>"]
+license: "MIT"
+homepage: "https://github.com/roddyyaga/ppx_rapper"
+doc: "https://github.com/roddyyaga/ppx_rapper"
+bug-reports: "https://github.com/roddyyaga/ppx_rapper/issues"
+depends: [
+  "ocaml" {>= "4.07"}
+  "dune" {>= "2.0.1"}
+  "pg_query"
+  "ppxlib"
+  "base"
```

Comment:
These two should most likely have a lower-bound constraint.
I would suspect something like
```
  "ppxlib" {>= "0.14.0"}
  "base" {>= "v0.9.0"}
```
or something like that, at the very least